client/components/course/Revisions/RevisionItem.vue

```diff
@@ -26,44 +29,36 @@ import {
   getRevisionAcronym,
   getRevisionColor
 } from 'utils/revision';
+import { mapActions, mapGetters } from 'vuex-module';
 import EntityRevisions from './EntityRevisions';
 import fecha from 'fecha';
 import find from 'lodash/find';
-import { mapGetters } from 'vuex-module';
 
 export default {
   name: 'revision-item',
   props: {
     revision: { type: Object, required: true }
   },
-  data() {
-    return { expanded: false };
-  },
+  data: () => ({ expanded: false }),
   computed: {
     ...mapGetters(['structure'], 'course'),
     ...mapGetters(['getParent'], 'activities'),
-    activity() {
```

Comment:
I don't see the benefit of going with the arrow function in this case.

Reply:
I used it for consistency, because other computed methods are arrow functions. Should I change it as it was before or?

Reply:
Actually not sure. I figured our convention is to use arrows where we can reduce the function to a one-liner and _normal_ otherwise.
@ExtensionEngine/tailor , thoughts?